session_0/word_processing.rb

```diff
@@ -18,9 +25,15 @@ def lower_case!(words)
 # words_with_prefix('apple', ['apple', 'ball', 'applesauce']) would
 # return the words 'apple' and 'applesauce'.
 def words_with_prefix(prefix, words)
-  raise NotImplementedError # TODO
+	match=[]
+	words.each do |w|
```

Comment:
Do you think we can use [select](https://www.rubyguides.com/2019/04/ruby-select-method/) here?

Reply:
I have made this change. Thank you for the suggestions.